miio/gateway/gateway_device/gateway_device.py

```diff
@@ -0,0 +1,32 @@
+"""Xiaomi Gateway device base class."""
```

Comment:
This can be moved to top level, rename to `gatewaydevice.py` to be more consistent with naming. Underscores are rather uncommon in python names.

Reply:
done